src/components/AddUsersModal/emailTemplate.js

```diff
@@ -0,0 +1,13 @@
+const emailTemplate = {
+  greeting: `Your Learning Manager has provided you with a license to give you access to take a course(s) on edX.
+`,
+  body: `You may redeem your license for any course(s) that are in your organization's subscription catalog. To redeem, you must [activate your license here](edx.org).
+
+edX Login: {USER_EMAIL}
```

Comment:
Note: slightly modified the email template to support the backend tokenization.

See https://github.com/edx/frontend-app-admin-portal/blob/a2a47d1320303a81ee70a3e2254b850ccd96d6d3/src/components/LicenseRemindModal/emailTemplate.js#L5